File: BoilerPlateObjc/ViewController.m
//
//  ViewController.m
//  BoilerPlateObjc
//
//  Created by Balaganesh on 04/06/22.
//

#import "ViewController.h"
#import <SwiftObjcFramework/SwiftObjcFramework-Swift.h>

@interface ViewController ()

@property PublicSwiftClass *swiftInstance;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    self.swiftInstance = [PublicSwiftClass new];
    [self.swiftInstance testSwiftFunction];
}


@end

File: BoilerPlateSwift/ViewController.swift
//
//  ViewController.swift
//  BoilerPlateSwift
//
//  Created by Balaganesh on 23/02/22.
//

import UIKit
import SwiftObjcFramework
//import SwiftObjcFramework_Private

class ViewController: UIViewController {

    let swiftInstance = PublicSwiftClass()
//    let objcInstance = InternalObjcClass()
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        swiftInstance.testObjcFunction()
//        objcClass.testFunction()
    }


}

